app/api/route/healthcheck.py

```diff
@@ -1,18 +1,18 @@
 from typing import Tuple
 
-import marshmallow
 from apiflask import APIBlueprint
 from sqlalchemy import text
 from werkzeug.exceptions import ServiceUnavailable
 
 import api.logging
 from api.route import response
 from api.route.api_context import api_context_manager
+from api.route.schemas import request_schema
 
 logger = api.logging.get_logger(__name__)
 
 
-class HealthcheckSchema(marshmallow.Schema):
+class HealthcheckSchema(request_schema.OrderedSchema):
```

Comment:
see OrderedSchema below in request_schema.py, this was needed to fix an issue where generated openapi schemas weren't deterministic